### ISSUE TITLE
Error when trying to create LDAP access server

### ISSUE THREAD

**Issue by mbrgm** [opnsense/core#1512]
Binding to the server works (checked by selecting the auth query from the GUI), but when I add the server I'm always getting

```
CSRF check failed. Your form session may have expired, or you may not have cookies enabled.
```

OPNsense 17.1-amd64
FreeBSD 11.0-RELEASE-p7
OpenSSL 1.0.2k 26 Jan 2017

**Comment by fichtner** [comment]
Please upgrade to 17.1.4, it has been fixed since :)

**Comment by AdSchellevis** [comment]
@mbrgm please upgrade, this one is solved in the current release.

**Comment by mbrgm** [comment]
Ok, I'll do this when my second instance is ready for failover. Feel
free to close this if it's a known issue anyway.

**Comment by fichtner** [comment]
Ok, closing. For reverence, it was fixed in 17.1.1:

https://github.com/opnsense/changelog/blob/051ca3cbe4052a7aefc579d3c58cf8c7c94b426e/doc/17.1/17.1.1#L17-#L18


Cheers,
Franco
